.travis.yml

```diff
@@ -7,7 +7,7 @@ env:
     - OCAML_VERSION=4.04.2
     - DISTRO=debian-stable
     - PACKAGE=xapi-nbd
-    - PINS="nbd:git://github.com/xapi-project/nbd.git#bugfix-v2.x"
+    - PINS="nbd:git://github.com/xapi-project/nbd.git#bugfix-v2.x xen-api-client:git://github.com/xapi-project/xen-api-client#feature/CBT xen-api-client-lwt:git://github.com/xapi-project/xen-api-client#feature/CBT xen-api-client-async:git://github.com/xapi-project/xen-api-client#feature/CBT"
```

Comment:
Does the syntax allow line-breaks within the list? It would be more readable (and maintainable) with one item per line.

Reply:
I'm not sure, but that's a good point. I'll try to figure it out when the Travis build passes.